CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## [1.0-beta-7] - 2019-02-20
+
+Bugfixes and improvements:
+
+- [[JENKINS-54352](https://issues.jenkins-ci.org/browse/JENKINS-54352)] Add functional tests
+- [[JENKINS-56079](https://issues.jenkins-ci.org/browse/JENKINS-56079)] Print the classpath in case the wrong directories are provided to Bootstrap
+
+Internal changes:
+
+* [[PR #26](https://github.com/jenkinsci/jenkinsfile-runner/pull/26)] Allow to pass worfklow job parameters
```

Comment:
It is not an internal change. It is actually a major feature. Also `--no-sandbox` could be mentioned